bin/kano-settings-onboot

```diff
@@ -205,7 +267,7 @@ def compare_and_set_optimal_resolution():
             if mode['mode'] == preferred_mode and mode['group'] == preferred_group:
                 preferred_aspect_ratio = mode['aspect']
                 logger.info('Reported preferred mode is {} group {} aspect ratio: {}'.format(
-                        preferred_mode, preferred_group, preferred_aspect_ratio))
+                    preferred_mode, preferred_group, preferred_aspect_ratio))
```

Comment:
Some weird stuff with parentheses matching making it difficult to follow. Probably should newline the matching `)`s, maybe something like

```python
logger.info(
    'Reported preferred mode is {} group {} aspect ratio: {}'.format(
        preferred_mode, preferred_group, preferred_aspect_ratio
    )
)
```
or
```python
logger.info(
    'Reported preferred mode is {} group {} aspect ratio: {}'
    .format(preferred_mode, preferred_group, preferred_aspect_ratio)
)
```